index.js

```diff
@@ -306,6 +306,7 @@ AFRAME.registerComponent('lounge-floor', {
     depth: {type: 'number', default: 7},
     color: {type: 'color', default: ''},
     texture: {type: 'asset', default: ''},
+    repeatTexture: {type: 'vec2', default: {x: 1, y: 1}},
```

Comment:
Since for now we only have the texture that can be repeated, and I don't envision anything else that could be repeated, would you mind using "repeat" instead of "repeatTexture"? This way it would also be consistent with the name used for the `material` property.